roles/testnode/tasks/nagios.yml

```diff
@@ -8,13 +8,19 @@
     mode: 0440
     validate: visudo -cf %s
 
-- name: Configure nagios nrpe settings.
-  template:
-    src: nagios/nagios-nrpe-server
-    dest: "/etc/default/{{ nrpe_service_name }}"
-    owner: root
-    group: root
-    mode: 0644
+- name: Configure nagios nrpe settings (Ubuntu)
+  lineinfile:
+    dest: /etc/default/{{ nrpe_service_name }}
+    regexp: "^DAEMON_OPTS.*"
```

Comment:
the .* isn't necessary, although it doesn't really hurt anything; it only has to match something on the line, not the whole line